.github/workflows/pypi.yaml

```diff
@@ -11,10 +11,10 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - run: python -m pip install --upgrade pip
-      - run: python -m pip install flake8 pytest
+      - run: python -m pip install flake8 pytest build
```

Comment:
The [build](https://pypi.org/project/build/) package seems to require Python 3.7:

> Requires: Python >= 3.7

Since Python < 3.7 isn't officially supported upstream, that's fine, but we should remove Python 3.3 to 3.6 from `.github/workflows/pytest.yaml`.

Probably off topic for this PR, but eventually we can think of migrating from `setup.py` to `pyproject.toml` ([docs](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/)).

Reply:
I've removed python 3.3 to 3.6 in the pytest part of the workflow.